docs/source/markdown/podman-import.1.md

```diff
@@ -30,10 +34,18 @@ Can be set multiple times
 
 Set commit message for imported image
 
+#### **--os**
+
+Set OS of the imported image.
+
 #### **--quiet**, **-q**
 
 Shows progress on the import
 
+#### **--variant**
+
+Set variant of the imported image.
+
```

Comment:
ditto valid values

Reply:
@TomSweeneyRedHat User is free to set any value if he is doing it manually however it's user's responsibility to make it work if they choose a wrong value.